src/arch/xtensa/include/arch/spinlock.h

```diff
@@ -118,10 +118,7 @@ static inline void arch_spin_unlock(struct k_spinlock *lock)
  */
```

Comment:
proper git summary prefix please

Reply:
@saisurya-ch Can you update the git commit message to have proper prefix on the first line?